### PR TITLE
Fix debate edit dates issue

### PR DESCRIPTION
**Pull request by Leusev**
#### :tophat: What? Why?

When editing a existing debate with Start / End dates, and only wanted to change one of the dates, the form validator threw an error because the other unchanged date field wasn't in wrong format, so form builder couldn't assign it to form.
This causes that this field was unassigned to form, so when debate's form validation field checked it, threw and error.

#### :clipboard: Subtasks
- [ ] Add `CHANGELOG` entry
- [ ] Add documentation regarding the feature 
- [ ] Add/modify seeds
- [ ] Add tests
- [ ] Another subtask

### CODE REVIEW

File: docs/HOW_TO_UPGRADE.md [hunk 1]
```diff
@@ -23,8 +23,6 @@ bin/rails db:migrate
 ```
 
   Custom modules with migrations.
-  * Decidim::Department ("decidim-department")
```
Comment (tramuntanal) [inline docs/HOW_TO_UPGRADE.md:26]: why remove this?

Comment (Leusev) [reply docs/HOW_TO_UPGRADE.md]: Right! I've fixed it, sorry!

**Comment by Leusev** [comment]
Hi @spuigmitjadxc 
How do you want us to carry out the deployment? Do you want us to do it first in staging to be able to test it?
Thanks in advance
// cc @tramuntanal

**Comment by spuigmitjadxc** [comment]
Hi @Leusev ,
Yes please, can you make the deploy and check that in production works well?
Thank you

**Pull request merged**